content/blog/2024-08-08-bootc-almalinux-heliumos.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Bringing bootc to AlmaLinux"
+type: blog
+author: 
+ name: "Isaac Beverly"
+ bio: "Developer, HeliumOS"
+ image: /users/imbev.jpg
+date: 2024-08-08
+images:
+  - /blog-images/2024/2024-08-08-bootc-almalinux.png
+post: 
+    title: "Bringing bootc to AlmaLinux"
+    image: /blog-images/2024/2024-08-08-bootc-almalinux.png
+---
+
+bootc is a new deployment method that powers [Image mode for RHEL](https://www.redhat.com/en/about/press-releases/red-hat-reimagines-enterprise-linux-ai-future-image-mode-red-hat-enterprise-linux) and [HeliumOS](https://www.heliumos.org/), a new Linux distribution. We are excited to announce that AlmaLinux now offers [bootc images for Intel/AMD(x86_64)](https://quay.io/repository/almalinuxorg/almalinux-bootc?tab=tags).
+
+## How did this start?
+My goal as developer of HeliumOS is to produce a Linux-based desktop operating system that is user-friendly, stable, and reliable. The first foray towards this goal used a Debian base, however this had some issues:
+
+- Debian has 3-5 years of support. Though stable relative to most Linux distributions, I wanted more stability.
+- An unexpected interruption during a traditional Linux package manager update can render the system un-bootable or otherwise broken.
+
+2 months ago, I learned of Image mode for RHEL, a new method of deploying RHEL that used OCI images for updates. Rebasing on Enterprise Linux and using bootc, the technology behind Image mode for RHEL would solve both of the issues that I found with Debian. After various experiments, HeliumOS [announced](https://www.heliumos.org/blog/post/heliumos-v9-alpha-is-available-for-downlaod/) an Alpha release based on CentOS Stream.
```

Comment:
Seems odd to advertise a CentOS Stream derivative on the AlmaLinux blog.

Reply:
The CentOS Stream derivative is now an AlmaLinux derivative.

Reply:
Marking resolve in lieu of the other thread.